src/screens/OrgList/OrgList.test.tsx

```diff
@@ -340,4 +343,114 @@ test('Search bar filters organizations by name', async () => {
   userEvent.clear(searchBar);
   userEvent.type(searchBar, 'Aka');
   expect(container.textContent).toMatch('Akatsuki');
+
+  // Test that the search bar filters all organization if there are is no search passed
+  userEvent.clear(searchBar);
+  userEvent.type(searchBar, '');
+  expect(container.textContent).toMatch('');
+});
+
+describe('SuperDashListCard', () => {
+  it('renders correctly when user type is SUPERADMIN', async () => {
+    localStorage.setItem('UserType', 'SUPERADMIN');
+    const datas = {
+      _id: '123',
+      image: 'https://example.com/image.png',
```

Comment:
We need to be using place holder images that are a part of our code base. Relying on external sources that could disappear is not a good practice.